### PR TITLE
Ensure importing another config file overwrites previous loaded variables

### PR DESCRIPTION
**Pull request by gandarez**

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -38,6 +38,8 @@ ssl_certs_file =
 timeout = 30
 hostname = machinename
 log_file =
+import_cfg = /path/to/another/wakatime.cfg
+```
```
Comment (alanhamlett) [inline USAGE.md]: A typo?

Comment (gandarez) [reply USAGE.md]: This setting wasn't documented so I added in this PR.

Comment (alanhamlett) [reply USAGE.md]: The "```" shouldn't be there

Comment (gandarez) [reply USAGE.md]: Oops!

**Review by alanhamlett — APPROVED** [review]

**Pull request merged**